src/ui/popup/manage-settings-page/export-theme.tsx

```diff
@@ -0,0 +1,34 @@
+import {m} from 'malevic';
+import {ViewProps} from '../types';
+import {Button} from '../../controls';
+import {saveFile} from '../../utils';
+import ControlGroup from '../control-group';
+
+export default function ExportTheme(props: ViewProps) {
+    const listener = ({type, data}) => {
+        if (type === 'export-css-response') {
+            saveFile('Dark-Reader-Settings.css', data);
```

Comment:
This file name is strange. Let's make it something like `DarkReader-www-google-com.css`. To achieve this you can make message data to be like `{host: string; css: string}` or as far as I remember the second argument for a message handler is `chrome.runtime.MessageSender`. You can get sender tab URL like `sender,tab.url` and then use a RegExp to get a host name. All `/[^a-z0-1\-]/g` characters should be replaced with `-`.

Reply:
`props.tab.url;`?